tests/cli/test_rasa_data.py

```diff
@@ -152,6 +153,87 @@ async def mock_from_importer(importer: TrainingDataImporter) -> Validator:
         data.validate_files(args)
 
 
+def test_rasa_data_augment_nlu(run: Callable[..., RunResult],):
```

Comment:
is there a way to speed this one up a bit? 40 seconds is a bit much for a single test. Also note that we don't test any edge cases (invalid file paths, invalid config)

Reply:
I don't think we can speed it up by much, it runs the rasa train/test loop twice, so that takes the bulk of the time. I guess I can strip out some opts from the test config and reduce the number of epochs for DIET even further (currently at `3`).  That will reduce the running time at least a bit.

Reply:
that would be great